pennylane/transforms/qmc.py

```diff
@@ -39,19 +43,24 @@ def _apply_controlled_z(wires, control_wire, work_wires):
         work_wires (Wires): the work wires used in the decomposition
     """
     target_wire = wires[0]
-    PauliX(target_wire)
-    Hadamard(target_wire)
+    updated_operations = []
```

Comment:
The docs for both this function as well as `apply_controlled_q` say that this function "Provides the circuit to apply ..." and it "performs the multi-controlled-Z". Since operations aren't supposed to be queued anymore, I think the wording should be updated.

Reply:
Updated, let me know if it reads better